examples/datasets/plot_opm_rest_data.py

```diff
@@ -0,0 +1,209 @@
+# Make sure our assumptions later hold
+assert raws['opm'].info['sfreq'] == raws['vv'].info['sfreq']
+
+##############################################################################
+# Do some minimal artifact rejection just for VectorView data
+
+titles = dict(vv='VectorView', opm='OPM')
+ssp_ecg, _ = mne.preprocessing.compute_proj_ecg(
+    raws['vv'], tmin=-0.1, tmax=0.1, n_grad=1, n_mag=2)
+raws['vv'].add_proj(ssp_ecg, remove_existing=True)
+ssp_ecg_eog, _ = mne.preprocessing.compute_proj_eog(
+    raws['vv'], n_grad=1, n_mag=1, ch_name='MEG0112')
+raws['vv'].add_proj(ssp_ecg_eog, remove_existing=True)
+raw_erms['vv'].add_proj(ssp_ecg_eog)
+fig = mne.viz.plot_projs_topomap(raws['vv'].info['projs'][-5:],
+                                 info=raws['vv'].info)
+fig.suptitle(titles['vv'])
+fig.subplots_adjust(0.05, 0.05, 0.95, 0.85)
+
+##############################################################################
+# Explore data
+
+kinds = ('vv', 'opm')
+n_fft = next_fast_len(int(round(4 * new_sfreq)))
+print('Using n_fft=%d (%0.1f sec)' % (n_fft, n_fft / raws['vv'].info['sfreq']))
+for kind in kinds:
+    fig = raws[kind].plot_psd(n_fft=n_fft, proj=True)
+    fig.suptitle(titles[kind])
+    fig.subplots_adjust(0.1, 0.1, 0.95, 0.85)
+
+##############################################################################
+# Alignment and forward
+# ---------------------
+
+src = mne.read_source_spaces(src_fname)
+bem = mne.read_bem_solution(bem_fname)
+fwd = dict()
+trans = dict(vv=vv_trans_fname, opm=opm_trans_fname)
+# check alignment and generate forward
+with mne.use_coil_def(opm_coil_def_fname):
+    for kind in kinds:
+        dig = True if kind == 'vv' else False
+        fig = mne.viz.plot_alignment(
+            raws[kind].info, trans=trans[kind], subject=subject,
+            subjects_dir=subjects_dir, dig=dig, coord_frame='mri',
+            surfaces=('head', 'white'))
+        mlab.view(0, 90, focalpoint=(0., 0., 0.), distance=0.6, figure=fig)
+        fwd[kind] = mne.make_forward_solution(
+            raws[kind].info, trans[kind], src, bem, eeg=False, verbose=True)
+
+##############################################################################
+# Compute and apply inverse to PSD estimated using multitaper + Welch.
+# Group into frequency bands, then normalize each source point and sensor
+# independently. This makes the value of each sensor point and source location
+# in each frequency band the percentage of the PSD accounted for by that band.
+
+freq_bands = dict(
+    delta=(2, 4), theta=(5, 7), alpha=(8, 12), beta=(15, 29), gamma=(30, 45))
+topos = dict(vv=dict(), opm=dict())
+stcs = dict(vv=dict(), opm=dict())
+
+for kind in kinds:
+    noise_cov = mne.compute_raw_covariance(raw_erms[kind])
+    inverse_operator = mne.minimum_norm.make_inverse_operator(
+        raws[kind].info, forward=fwd[kind], noise_cov=noise_cov, verbose=True)
+    stc_psd, evoked_psd = mne.minimum_norm.compute_source_psd(
```

Comment:
it is confusing that this is called evoked_psd. I'd say average_psd or sensor_psd.

Reply:
or ch_psd

Reply:
Can we please reconsider this? I think we should not do this as new comers come from textbooks and papers. Evoked means timelocked / ERP / ERF and if we have bad habits like calling data evoked because they are stored in an evoked class we should not continue with them in new PRs. We may want to rename evoked class or make a parent class called Average from which we derive Evoked and PSDAverage.